File: dart-jaguar-client/lib/model/fields_edge_module_object.dart
import 'package:jaguar_serializer/jaguar_serializer.dart';

import 'package:swagger/model/field_object_fields.dart';
part 'fields_edge_module_object.jser.dart';

class FieldsEdgeModuleObject {
  
  final String name;
  
  final List<FieldObjectFields> fields;
  
  final bool custom;
  

  FieldsEdgeModuleObject(
    

{
    
     this.name = null,   this.fields = const [],  
     this.custom = null 
    
    }
  );

  @override
  String toString() {
    return 'FieldsEdgeModuleObject[name=$name, fields=$fields, custom=$custom, ]';
  }
}

@GenSerializer()
class FieldsEdgeModuleObjectSerializer extends Serializer<FieldsEdgeModuleObject> with _$FieldsEdgeModuleObjectSerializer {

}

File: scala-gatling-client/src/gatling/scala/io/swagger/client/model/FieldsEdgeModuleObject.scala

package io.swagger.client.model


case class FieldsEdgeModuleObject (
    _name: String,
    _fields: Option[List[FieldObject_fields]],
    _custom: Option[Boolean]
)
object FieldsEdgeModuleObject {
    def toStringBody(var_name: Object, var_fields: Object, var_custom: Object) =
        s"""
        | {
        | "name":$var_name,"fields":$var_fields,"custom":$var_custom
        | }
        """.stripMargin
}
